mix.exs

```diff
@@ -37,7 +42,9 @@ defmodule Feedback.Mixfile do
      {:phoenix_html, "~> 2.6"},
      {:phoenix_live_reload, "~> 1.0", only: :dev},
      {:gettext, "~> 0.11"},
-     {:cowboy, "~> 1.0"}]
+     {:cowboy, "~> 1.0"},
+     {:comeonin, "~> 2.0"},
```

Comment:
@nelsonic we'll need it for a user to log in because we require the `checkpw` function it provides